tests/www/companies_views/test_edit_company_views.py

```diff
@@ -91,7 +89,7 @@ def test_edit(client, mocker):
     # Go forward again
     response = client.post(step_2_url, data=post_data, follow=True)
     attrs = 'target="_blank" rel="noopener" aria-label="Ouverture dans un nouvel onglet"'
-    assertContains(response, "Aperçu de la fiche")
+    assertContains(response, "Aperçu")
```

Comment:
Ajouter un peu de contexte permet d’éviter un faux positif.
```suggestion
    assertContains(response, "<legend class="h3">Aperçu</legend>")
```